docs/_inc/_install-python.md

```diff
@@ -1,6 +1,6 @@
 Installing Python is beyond the scope of this documentation.
 However, it is recommended to use a Python version manager, {term}`pyenv`, that allows you to install multiple versions of Python on your development environment without destroying your system's Python.
 % TODO: uncomment this line after upgrading to plone-sphinx-theme and latest Sphinx which supports replacements inside includes.
-% Plone requires Python version {SUPPORTED_PYTHON_VERSIONS}.
+% Plone requires Python version {SUPPORTED_PYTHON_VERSIONS_PLONE60}.
 
 Plone requires Python version 3.8, 3.9, 3.10, 3.11, or 3.12.
```

Comment:
@stevepiercy Shouldn't we specify like this?

Plone 6.0 requires Python version 3.8, 3.9, 3.10, 3.11, or 3.12.
Plone 6.1 requires Python version 3.10, 3.11, or 3.12.